src/botmsg.c

```diff
@@ -757,8 +757,9 @@ void botnet_send_nkch_part(int butidx, int useridx, char *oldnick)
  */
 int add_note(char *to, char *from, char *msg, int idx, int echo)
 {
+  #define FROMLEN 40
```

Comment:
HANDLEN (or something larger than HANDLEN) would be the correct value, right?

Reply:
`NOTENAMELEN` as used by `.tcl sendnote`, though `.tcl storenote` (from the notes.mod) allows for even bigger (`sizeof argv[1]`!).